cmake/parflow_config.h.in

```diff
@@ -47,6 +47,7 @@
 #cmakedefine HAVE_SLURM
 
 #cmakedefine PARFLOW_HAVE_NETCDF
+#cmakedefine HAVE_NETCDF
```

Comment:
I don't see the HAVE_NETCDF symbol being used, was it needed by something?   The goal it so remove the un-prefixed names to make the macro definitions in a PARFLOW_ namespace so we do not get collisions with other includes.   We did not do this originally for all the preprocessor macros but all ones should have a prefix on them.

Everything else looks OK, thanks for the patch.  I'm massively confused why I was able to build with CMake and netcdf with these issues.

Reply:
you're right. I fixed the HAVE_NETCDF issue.